rholang/src/main/scala/coop/rchain/rholang/interpreter/Runtime.scala

```diff
@@ -99,10 +99,10 @@ object Runtime {
   type RhoSysFunction    = Function1[Seq[ListParWithRandomAndPhlos], Task[Unit]]
   type RhoDispatchMap    = Map[Long, RhoSysFunction]
 
-  private type CPAK[F[_, _, _, _]] =
+  type CPAK[F[_, _, _, _]] =
```

Comment:
Did you try making it protected? If that doesn't work too, please at least leave a comment saying it should not be used otherwise than for defining other type aliases in this object